src/esp/bindings/CMakeLists.txt

```diff
@@ -36,6 +36,17 @@ target_link_libraries(
          sim
 )
 
+if(BUILD_WITH_AUDIO)
+  add_custom_command(
```

Comment:
Really wish I could find a nice way to copy around this binary file. Also this code is hard coded to only support Linux at the moment (.so) extension.

Reply:
Wait, didn't my suggestion use `find_library()`? *That* would be a portable way to do this.

Reply:
Isnt that supposed to be part of sensor's CmakeList.txt?
https://github.com/facebookresearch/habitat-sim/blob/sanchitgarg/audio-sensor/src/esp/sensor/CMakeLists.txt#L56-L60

Reply:
Ahem, yes, there also. So, like, if you have it in one place already, why not reuse the found path in this place as well? Or, more robustly, do the library finding somewhere higher (say, in `dependencies.cmake`), and then use the `${HabitatAcoustics_LIBRARY}` variable in both places.